.github/workflows/deploy.yaml

```diff
@@ -8,20 +8,138 @@ on:
   workflow_dispatch:
```

Comment:
I thought it would be totally manual for now, maybe have it deploy to `dev` on merges to main eventually but I think many of our deployments will just be to check out a new version of the API repo so won't require any code changes.